test/integration/routes.test.js

```diff
@@ -52,44 +55,155 @@ describe('routes', function () {
   })
 
   describe('health check', function () {
-    let app, statusHandler
+    describe('happy path', function () {
+      let app, statusHandler
 
-    before(async function () {
-      const server = await createHttpServer()
-      app = server.app
-      statusHandler = server.statusHandler
+      before(async function () {
+        const server = await createHttpServer()
+        app = server.app
+        statusHandler = server.statusHandler
+      })
+
+      after(function () {
+        statusHandler.close()
+      })
+
+      test('health check', async function () {
+        const expectedResult = {
```

Comment:
maybe expected results could be abstracted into fixtures to keep test scenarios clearer?

Reply:
yeah I'll move this out to an expectations file

Reply:
done